wyzesense2mqtt/wyzesense.py

```diff
@@ -305,6 +322,46 @@ def _OnEventLog(self, pkt):
         msg = pkt.Payload[9:]
         log.info("LOG: time=%s, data=%s", tm.isoformat(), bytes_to_hex(msg))
 
+    def _OnHMSEvent(self, pkt):
+        payloadSubType = pkt.Payload[0x0E]
+        if payloadSubType == 0x12:
+            #Water sensor event
+            typeByte, mac = struct.unpack_from(">B8s", pkt.Payload); 
+            mac = mac.decode('ascii')  
+            water = pkt.Payload[0x0F]
+            ext_water = pkt.Payload[0x10]
+            has_ext = pkt.Payload[0x11]
+            signal = pkt.Payload[0x14]
+            battery = pkt.Payload[0x0C]
+            e = SensorEvent(mac, datetime.datetime.utcnow(), "water", (water, ext_water, has_ext, battery, signal))
+            self.__on_event(self, e)
+        elif payloadSubType == 0x0A or payloadSubType == 0x02:
+            typeByte, mac = struct.unpack_from(">B8s", pkt.Payload); 
+            mac = mac.decode('ascii')  
+            battery = pkt.Payload[0x0C]
```

Comment:
This one is consistently (for me anyways) a value of around 150... How can we figure the actual battery value from that?